scripts/wcurl

```diff
@@ -91,7 +91,7 @@ error()
 
 # Extra curl options provided by the user.
 # This is set per-URL for every URL provided.
-# Some options are global, but we are erroring on the side of needlesly setting
+# Some options are global, but we are erroring on the side of needlessly setting
```

Comment:
We import wcurl, please submit this one in wcurl's separate tracker at https://github.com/curl/wcurl/issues